src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="home">
+        <a href="index.html">
+          <img class="logo"
+          src="images/logo.png"
+          alt="MoYo_logotype">
```

Comment:
please fix the code styling according to the checklist. fix all places in HTML file
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/cd66e129-eeae-43a2-aeed-8eb9adce8365)